### PR TITLE
fix MeshNetworks demo

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: mesh/v1alpha1/network.proto [hunk 1]
```diff
@@ -35,7 +35,7 @@ message Network {
   //
   // 2. Explicitly:
   //
-  //    a. By matching the registry name with one of the "from_registries"
+  //    a. By matching the registry name with one of the "from_registry"
```
Comment (rshriram) [inline mesh/v1alpha1/network.proto]: fromRegistry (we refer to the YAML form in docs)

Comment (hzxuzhonghu) [reply mesh/v1alpha1/network.proto]: ACK

**Pull request merged**